README.md

```diff
@@ -62,6 +62,7 @@ Command                          | Description
 `-w`, `--workers <n>`            | The number of worker threads to spawn. The default is 0.
 `-m`, `--mirror [host:port]`     | Mirror transactions to another cache server. Repeat this option ofr multiple mirrors.
 `-m`, `--monitor-parent-process <n>` | Monitor a parent process and exit if it dies.
+`--allow-ipv6`                  | Allow IPv6 connections if available.
```

Comment:
I would prefer if this were in the config file - we need to move toward less discrepancy between CLI options and config file options.

To do so, I think a new top level section called "Server" would be appropriate:

```
Server
   options:
      allowIpv6: true
```